style.css

```diff
@@ -0,0 +1,23 @@
+#title {
+  text-align: center;
+}
+#survey-form {
```

Comment:
I think you should try to follow the semantics of CSS and give new line between two different selectors.

Reply:
[OPTIONAL] I think you should work more on giving valid reasons for your suggestions. Spacing out CSS rules isn't for semantic reasons. What do you think:thinking: